Languages/german.lang

```diff
@@ -410,7 +410,7 @@ msgid "Cheatfile is blank"
 msgstr "Cheatdatei ist leer"
 
 msgid "Clear"
-msgstr "Freigeben"
+msgstr "Löschen"
```

Comment:
Please check again the context where `Clear` ist used. IMHO the translation is correct.

Reply:
"clear" is used in the GUI settings to clear the input. I can send a screenshot if you need one.

Reply:
> Please check again the context where `Clear` is used. IMHO the translation is correct.

Here is the screenshot:
![2023-07-07-13-35-04-054](https://github.com/wiidev/usbloadergx/assets/94064167/8bea0a63-cad5-4895-9d60-f06bc8ad2906)

Reply:
I think "Freigeben" does not make any sense in this context.

Reply:
@eku 
Could you maybe commit my suggestion?